Update latex language grammar and configuration

jlelong opened the pull request:
This PR does 3 things

1. Update the LaTeX language configuration to add big delimiters with `|`, `\|`, `[lr][vV]ert`, `\langle...\rangle` to bracket pairs. Related to 

    - jlelong/vscode-latex-basics#11
    - jlelong/vscode-latex-basics#13

2. Use the new `markdown` language configuration for  the combined `markdown-latex` language
3. Update the grammar files

    - jlelong/vscode-latex-basics#15

@aeschli I guess this PR is for you. I hope I did it right.

alexr00 reviewed (approved):
Thank you for the pull request! I've just updated the cgmanifest.json file. 

For updating the syntax files: I usually run the script that updates them once per release. You are always welcome to make PRs to update them too, but if you don't feel like it, then I'll update them eventually too!

jlelong commented:
> Thank you for the pull request! I've just updated the cgmanifest.json file.

I thought I had updated it but I probably lost it somewhere. Thanks for taking care of this!
 
> For updating the syntax files: I usually run the script that updates them once per release. You are always welcome to make PRs to update them too, but if you don't feel like it then I'll update them eventually too!

Yes. However, the build script only updates the grammar files but not the language configuration files. Hence, this PR. Basically, I will only make a PR when those language configuration files need to be updated.
